src/openforms/submissions/models/submission.py

```diff
@@ -241,6 +242,15 @@ class Submission(models.Model):
         "submissions.Submission", on_delete=models.SET_NULL, null=True, blank=True
     )
 
+    language_code = models.CharField(
+        _("language code"),
+        max_length=2,
+        default=settings.LANGUAGE_CODE,
```

Comment:
Not sure if we want a default or if we want it to be explicitly filled (with a data migration to account for existing `Submission`s)

Reply:
Django might populate existing data with the `default`, but I'll have to check.

Old submissions are pruned anyway, so I think we can leave it empty. This would then only apply to the edge case where the language code is included to the registration backend and has an empty value, so as long as we have an explicit test case for that behaviour, it should be fine.

Reply:
So no default and set `blank=True`? What would we want to test in that test case, that it is not possible to complete a submission with an empty language code?